cmds/pflask/pflask.go

```diff
@@ -373,14 +375,14 @@ func main() {
 	MountAll(*chroot, unprivileged)
 
 	if !unprivileged {
-		copy_nodes(*chroot)
+		copyNodes(*chroot)
 	}
 
-	make_ptmx(*chroot)
+	makePtmx(*chroot)
```

Comment:
Note that all of these could be changed to use the support code in uroot/root.go. But it's fine for now. This originally came from minix and you can tell how old it is by all the non-standard things like _, etc.

Reply:
Good to know, thanks. Will send a follow-up PR to use the code from `uroot/root.go`